src/fsharp/infos.fs

```diff
@@ -3035,6 +3038,14 @@ type PropertyCollector(g,amap,m,typ,optFilter,ad) =
 /// Query the immediate properties of an F# type, not taking into account inherited properties. The optFilter
 /// parameter is an optional name to restrict the set of properties returned.
 let GetImmediateIntrinsicPropInfosOfType (optFilter,ad) g amap m typ =
+    let getFSharpMetadata() =
```

Comment:
Generally this code avoids inner functions - just create a new top level function ``GetImmediateIntrinsicFSharpPropInfosOfType`` please